c-deps/libroach/ccl/key_manager.h

```diff
@@ -85,8 +87,6 @@ class FileKeyManager : public KeyManager {
                           const std::string& old_key_path)
       : env_(env), active_key_path_(active_key_path), old_key_path_(old_key_path) {}
 
-  virtual ~FileKeyManager();
```

Comment:
That should stay for any direct users of `FileKeyManager`

Reply:
No it doesn't: the parent class of `FileKeyManager` already has a virtual destructor (with my change above), so automatically all derived classes have their destructor virtual too.

Reply:
Sure, but it's still weird to have `~FileKeyManager()` in the `.cc` that isn't reflected in the `.h`.

Reply:
oh I see, sorry for that. I am so used to template programming with all code in headers I hadn't even looked at the cc file. 

Fixed, PTAL.